coreferee/lang/fr/language_specific_rules.py

```diff
@@ -264,7 +264,7 @@ def is_potential_anaphor(self, token: Token) -> bool:
             and token.dep_ != "expl:subj"
         ):
             return True
-        if token.pos_ == "DET" and token.dep_ != "det":
+        if token.pos_ == "DET" and token.dep_ != "det": # Example
             return True
         if not (
             (
```

Comment:
Here an example (with the `_md` model only) is `Je la previens`. `la` gets `pos_ == "DET"` and `dep_ == "obj"`.

Reply:
Oh, it's a problem that had been bugging me for a while.
I hadn't considered doing this aha, brillant idea!

Are we sure that it does not cause some actual (not mistagged) DETs to be considered anaphors ? I'll check this